### PR TITLE
Add `UseEscapeHatch` and replace the existing `UseEscapeHatch` with `EnableEscapeHatch`

### PR DESCRIPTION
**Pull request by Sneh1999**
This PR separates the `UseEscapeHatch` and `EnableEcapeHatch`

### CODE REVIEW

**Comment by ImJeremyHe** [comment]
Would you like to elaborate a bit about this PR? @Sneh1999 
I think if the node is always waiting for HotShot going live, this actually means it doesn't have an "escape hatch" at all. So I think one flag is enough.

**Comment by Sneh1999** [comment]
> Would you like to elaborate a bit about this PR? @Sneh1999 I think if the node is always waiting for HotShot going live, this actually means it doesn't have an "escape hatch" at all. So I think one flag is enough.

Yes feel free to make changes, it’s an incomplete PR

File: arbnode/transaction_streamer.go [hunk 1]
```diff
@@ -1707,67 +1709,37 @@ func (s *TransactionStreamer) submitEspressoTransactions(ctx context.Context) ti
...
-	l1Height := header.Header.GetL1Head()
-	hotshotLive, err := s.lightClientReader.IsHotShotLiveAtHeight(l1Height, s.espressoSwitchDelayThreshold)
-	if err != nil {
-		return err
-	}
-	if hotshotLive {
-		// This transaction will be still finalized
-		return nil
-	}
+	// Skip the espresso verification for the submitted messages
```
[the remark this answers is not in the document]
Comment (Sneh1999) [reply arbnode/transaction_streamer.go:1736]: Yea that sounds good, if escape hatch is turned on and hotshot is down we can skip verification. @ImJeremyHe feel free to remove this!

File: arbnode/batch_poster.go [hunk 2]
```diff
@@ -183,7 +183,6 @@ type BatchPosterConfig struct {
 	EspressoTxnsPollingInterval  time.Duration `koanf:"espresso-txns-polling-interval"`
 	EspressoSwitchDelayThreshold uint64        `koanf:"espresso-switch-delay-threshold"`
 	EspressoMaxTransactionSize   uint64        `koanf:"espresso-max-transaction-size"`
-	EspressoTEEVerifierAddress   string        `koanf:"espresso-tee-verifier-address"`
```
Comment (ImJeremyHe) [inline arbnode/batch_poster.go:186]: This seems unnecesary for the batch poster.

We can use `lightClientAddress` and `espressoClient` to check if we are using espresso mode

**Review by Sneh1999 — COMMENTED** [review]
Thanks @ImJeremyHe, I really appreciate your help!

**Review by ImJeremyHe — APPROVED** [review]

**Pull request merged**